Makefile

```
@@ -1,9 +1,8 @@
 TEST?=$$(go list ./... |grep -v 'vendor')
 GOFMT_FILES?=$$(find . -name '*.go' |grep -v vendor)
-WEBSITE_REPO=github.com/hashicorp/terraform-website
+WEBSITE_REPO=terraform-website
```

Is there a reason for changing it from the repo path on github to just the name of the repo? It is a bit less clear where we are pulling it from now.
```suggestion
WEBSITE_REPO=hashicorp/terraform-website
```